x/adminmodule/types/keys.go

```diff
@@ -23,12 +23,17 @@ const (
 	AdminKey = "Admin-"
 
 	ArchiveKey = "Archive-"
+
+	ArchiveLegacyKey = "ArchiveLegacy-"
 )
 
 var (
-	ProposalsKeyPrefix        = []byte{0x00}
-	ActiveProposalQueuePrefix = []byte{0x01}
-	ProposalIDKey             = []byte{0x03}
+	ProposalsKeyPrefix              = []byte{0x00}
+	ProposalsKeyLegacyPrefix        = []byte{0x01}
+	ActiveProposalQueuePrefix       = []byte{0x02}
+	ActiveProposalLegacyQueuePrefix = []byte{0x03}
+	ProposalIDKey                   = []byte{0x04}
+	ProposalIDKeyLegacy             = []byte{0x05}
```

Comment:
These changes will break the state since you change keys but you don't migrate the storage.
Either implement a migration of the storage, or leave the old keys unchanged